database/dummy.js

```diff
@@ -0,0 +1,50 @@
+const faker = require('faker');
+const momentRandom = require('moment-random');
+
+const restaurantName = ['A Salt & Battery', 'Party Fowl', 'Thai Me Up Restaurant & Brewery', 'Wild Thyme Café', 'Tequila Mockingbird', 'Life of Pie', 'Sconehenge', 'The Codfather', 'The Notorious P.I.G', 'Pig’N Pancake', 'Backyard Bowls', 'The Little Easy', 'Pies n’ Thighs', 'Pot Au Feu', 'Cibo Matto', 'Siena', 'The Dorrance', 'The French Laundry', 'Dining Divas', 'Kitchen Corral', 'Uzen', 'Crystal Fish', 'Okaine', 'Chicken Things', 'Acapulco Gold', 'Achilles', 'Acorn Palace', 'Affecté', 'Affluential', 'Bleeko', 'Big Juicy', 'Bella Figala', 'Cafe Constant', 'Capricorn', 'Club Soma', 'Classio', 'Crab and Prawn', 'Crustacia', 'Dutch Oven Gretel', 'Eat Underground', 'Escargot', 'Food Frolic', 'Funky Chicken Soup', 'Gelatosphere', 'Godzilla Pizza', 'Homeroom Cafe', 'Hot Sour and Soar', 'Kaku', 'La Zilla', 'Mongoliant Little Sheep', 'Left Blank', 'Lizzie Bordens', 'Mint Dynasty', 'Mongerfish', 'Netplosion', 'New Orlean', 'Nibbles', 'Nob Hill', 'Nosh on the Net', 'Nostrilé', 'Passion Fruit', 'Peace Meal', 'Pastor Pizza', 'Radiant Fresh', 'Kurasabe', 'Raja', 'Rococo', 'Royal Lush', 'Saguaro', 'SeaSky', 'Shitaki', 'Seafood Seranade', 'Scrap Opera', 'Té Tango', 'Tequila Mockingbird', 'Thalia', 'The 4AM Club', 'The Beef Plank', 'The Crawfish Cantina', 'The Crustacean', 'The Potato Constant', 'Torpor Ritz', 'Troché', 'Waffle Stomper', 'Wasabi Fresh', 'Wire House', 'Zen BBQ', 'Zoo Stew', 'Zuke', 'Song Bird', 'Zilladelphia', 'Dinner in the Sky', ' Sur un Arbre Perché', 'Normas', ' Devil Island Prison Restaurant ', 'Ninja Dining', 'Aurum', 'Homewood Mountain Resort', 'Mountain Retreat', 'The Five Fishermen'];
```

Comment:
One thing to consider is using Faker to generate a random name. It may not be a restaurant name, but maybe it would be easier than writing a bunch of restaurant names. These names are cool though :)

Reply:
I added faker but couldn't find any appropriate category for restaurants name so I just manually picked 100 name for the dummy listing